crates/chia-datalayer/src/lib.rs

```diff
@@ -0,0 +1,3 @@
+mod merkle;
+
+pub use merkle::{InsertLocation, InternalNode, LeafNode, MerkleBlob, Side};
```

Comment:
Does this need to export more things? Like `TreeIndex`, `KvId`, `Block`, `Node`, etc? Might be simpler just to do `pub use merkle::*;`

Reply:
changed as suggested 6904bcd2cd1e3accc6bc8d2aef9ebb114412d514